Drop-In/src/test/java/com/braintreepayments/api/dropin/AddCardActivityUnitTest.java

```diff
@@ -975,10 +975,6 @@ public void showsSubmitButtonAgainWhenThreeDSecureIsCanceled() throws PackageMan
         setText(mEditCardView, R.id.bt_card_form_expiration, ExpirationDate.VALID_EXPIRATION);
         mEditCardView.findViewById(R.id.bt_button).performClick();
 
-        Intent nextStartedActivity = shadowOf(mActivity).peekNextStartedActivity();
```

Comment:
This `showsSubmitButtonAgainWhenThreeDSecureIsCanceled` unit test was failing after upgrading because of a null pointer on the `nextStartedActivity` here. I think it has something to do with testing library version and the shadow activity, but I couldn't pinpoint the issue. I'm not entirely sure what the value of these assertions are, and the test passed without them. I manually tested the 3DS cancel flow and verified that the button is visible. Are we ok with removing these two assertions?

Reply:
Yeah these assertions seem unrelated to the actual named test. I'd be cool with removing those too since we really want to know that the user can retry after a 3DS cancellation.